src/web/Content/client/dumpling.py

```diff
@@ -877,6 +880,16 @@ def _download_dump(self, dir, dumpManifest):
 
         return dumplingDir
     
+    def Hang(self, config):
+        if os.path.exists(config.dbgpath) and os.path.isdir(config.outpath):
+            CommandProcessor._create_hang_dump(config.pid, config.outpath, config.dbgpath)
+            process = psutil.Process(int(config.pid))
+            for child_process in process.children(recursive=True):
+                CommandProcessor._create_hang_dump(str(child_process.pid), config.outpath, config.dbgpath)
+        else:
+            path = config.dbgpath if os.path.exists(config.outpath) else config.outpath
```

Comment:
this should be os.path.isdir(config.outpath) not os.path.exists(config.outpath) I think.